news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest [Spotlight season] released very recently, we are underway reworking the osu! Spotlights from grounds up, starting with a brand new team and adding a whole new league system.
+
+![](image)
+
+Since March 2017 osu! Spotlights work in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea and give the curation and participation in the spotlights a whole new twist.
+While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
+
+We are still working on the details on how this league will work in detail, but here are some aspects we _plan_ to do:
+
+- **We want to host weekly timeshift lobbies**, using osu!lazer's recently implemented feature, having people compete actively on a portion of the beatmaps we have selected as Spotlight Beatmaps.
+- With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of that we will only publish portions of the whole spotlights every week.
+- When a timeshift lobby ends, we want to **award the winners of the timeshift lobbies weekly with a small prize**.
+- On top of that we also want to add a **Seasonal Ranking** where we will sum up the results of each timeshift lobby and pit every participant against another with a custom ranking system. We plan to assign a league bracket to every participant, based on their placement in this ranking system (Bronze, Silver, Gold, etc.). Once a season ends, maybe you can even keep your placement in shape of a medal or badge on your profile...?
+
+The aim of this league is to be an alternative, competitive and fun weekly activity for everyone to participate in while we use the maps pools to recommend people the best beatmaps osu! has to offer. This will give beatmaps in the Spotlight the attention they deserve even more.
```

Comment:
```suggestion
The aim of this league is to be an alternative, competitive and fun weekly activity for everyone to participate in while we use the mappools to recommend people the best beatmaps osu! has to offer. This will give beatmaps in the Spotlight the attention they deserve even more.
```